README.md

```diff
@@ -16,6 +16,44 @@ npm install @eslint/eslintrc --save-dev
 yarn add @eslint/eslintrc -D
 ```
 
+## Usage
+
+The primary class in this package is `FlatCompat`, which is a utility to translate ESLintRC-style configs into flat configs. Here's how you use it inside of your `eslint.config.js` file:
+
+```js
+import { FlatCompat } from "@eslint/eslintrc";
+
```

Comment:
is it native es modules?

we have to use cjs - util we can drop node < 12. https://github.com/eslint/eslint/issues/14023

Reply:
These config files support ESM. By the time this feature is finalized, we will have dropped Node.JS 10 support.

Reply:
Because we cannot use named imports on `"@eslint/eslintrc"` currently, maybe this example will confuse people. (People have to use the default import because this package uses CJS exports.)

Reply:
It doesn’t matter right now because no one is using the package besides us. We can always update this later if necessary. I just wanted to create a draft instead of having nothing.

Reply:
I clarified that this is future usage, not ready for right now.